tutorials/scripting/gdscript/gdscript_basics.rst

```diff
@@ -264,12 +266,8 @@ The following is the list of supported operators and their precedence.
 +--------------------------------------------------------------------------------+-------------------------------------------+
 | ``<`` ``>`` ``==`` ``!=`` ``>=`` ``<=``                                        | Comparisons                               |
 +--------------------------------------------------------------------------------+-------------------------------------------+
-| ``in``                                                                         | When used with the ``if`` keyword it      |
-|                                                                                | checks if a value is within a string,     |
-|                                                                                | list, range, dictionary, or node.         |
-|                                                                                | When used with the ``for`` keyword it is  |
-|                                                                                | used to iterate though the contents of a  |
-|                                                                                | string, list, range, dictionary or node.  |
+| ``in``                                                                         | Content test / Content iterator           |
+|                                                                                | (when used with ``for``)                  |
```

Comment:
I'd go with 

> Inclusion checker (when used with control flow keywords or in a standalone expression).
> Content iterator (when used with the ``for`` keyword).

Reply:
Thank you, i was struggling to use the right words about the subject